sensor_msgs/Image frame
---
bool updated